support-frontend/assets/pages/[countryGroupId]/checkout/components/PersonalDetailsFields.tsx

```diff
@@ -45,6 +55,9 @@ export function PersonalDetailsFields({
 					}}
 					onBlur={(event) => {
 						event.target.checkValidity();
+						if (email.length > 0) {
```

Comment:
Sorry my comment here might not have been clear https://github.com/guardian/support-frontend/pull/6751#discussion_r1936994386.

I was suggesting we check `confirmEmailRef.current?.querySelector('input')` has a value before checking it's validity, as I suspected we could have this scenario:

- Non-signed in user enters their email
- User tabs out of the email input and into the confirm email input
- The email onBlur immediately triggers a validation error on the empty confirm email input 

Does this happen?

Reply:
yes, sorry you're right that should be `confirmEmail.length`. Although that might be slightly confusing to mix together the react state variable and the event target